global.json

```diff
@@ -11,6 +11,6 @@
       }
```

Comment:
Do you still need the runtimes section?

Reply:
Yes, we need the runtimes section to get some bug fixes in the hosting APIs for components. If we didn't need these bug fixes (or if they were in .NET 5), we wouldn't have the section.